src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -5154,7 +5312,7 @@ end subroutine dist_km2deg
 !    
 !    implicit none
 !    
-!    integer, intent(in) :: N_ens, N_obs, N_catd
+!    integer, intent(in) :: N_ens, N_obs, N_catd 
```

Comment:
The new branch deletes a bunch of commented-out old code (Lines 5166-5221 in the develop branch).
But the clean-up isn't consistent.  I think we should leave this type of clean-up to a separate PR.  For now, it would be better to restore the deleted lines and focus on the snow DA changes.